src/domains/load-test/env/weu-dev/terraform.tfvars

```diff
@@ -23,5 +23,5 @@ terraform_remote_state_core = {
   key                  = "dev.terraform.tfstate"
 }
 
-dns_zone_prefix = "dev.platform"
+dns_zone_prefix = "dev.platform.pagopa.it"
```

Comment:
why in `dns_zone_prefix` y'd add `.pagopa.it` too ?

Reply:
because without it, it gives me an error